pkg/controller/system/system_controller.go

```diff
@@ -150,13 +151,18 @@ func (r *ReconcileSystem) installRootCertificates(instance *starlingxv1.System)
 			return err
 		}
 
-		caBytes, ok := secret.Data[starlingxv1.SecretCaCertKey]
-		if !ok {
-			// This can be valid as long as the target certificate is signed
-			// by a known CA certificate; otherwise once the target certificate
-			// is installed we will not be able to verify the TLS session.
-			log.Info("platform certificate without a CA certificate; ignoring", "name", c.Secret)
-			continue
+		var caBytes []byte
+		var ok bool
+		numRetries := 30
+		for iter := 0; iter < numRetries; iter++ {
+			caBytes, ok = secret.Data[starlingxv1.SecretCaCertKey]
+			if !ok {
+				log.Info("Platform certificate CA not ready/available", "name", c.Secret)
+				time.Sleep(5 * time.Second)
+			} else {
+				log.Info("Platform certificate CA ready!")
+				break
+			}
 		}
```

Comment:
Should there be a log to inform the user that DM will continue without a CA certificate if it times out?